templates/pages/reset-password.jsx

```diff
@@ -32,28 +35,52 @@ var ResetPassword = React.createClass({
     linkQuery.state = this.state.queryObj.state;
     linkQuery.scopes = this.state.queryObj.scopes;
     linkQuery.response_type = this.state.queryObj.response_type;
+
+    var wrapperClass = "centerDiv";
```

Comment:
possible nit: looks like this does the same class assignment as PasswordResetSuccess, worth refactoring to a utility function that both can call?